Minnet App/src/utils/constants/index.ts

```diff
@@ -0,0 +1,55 @@
+const path = "assets/icons";
+
+
+
+export const rows = [
+    {
```

Comment:
Try to give descriptive name

Reply:
gave proper descriptive name to the array of objects, resolved.